main.js

```diff
@@ -1,81 +1,99 @@
-var userChoice = prompt("Do you choose rock, paper or scissors?");
+  var userWin  = 0;
+  var computerWin = 0;
 
-var computerChoice = Math.floor(Math.random()*(10-1+1)+1);;
-if (computerChoice <3.33){
-    computerChoice = "rock";
-}else if(computerChoice <=5){
-    computerChoice = "paper";
-}
-else{
-    computerChoice = "scissors";
-}
+  function compare ()
+  {
+  choice1 = prompt("Do you choose rock, paper or scissors?");
 
-console.log(computerChoice);
-var userWin  = 0;
-var computerWin = 0;
+  choice2 = Math.floor(Math.random()*(10-1+1)+1);
+  if (choice2 <3.33)
```

Comment:
The Math.floor() function returns the largest integer, why do you need to check 3.33?